modules/core/src/main/scala/fs2/kafka/package.scala

```diff
@@ -257,7 +257,7 @@ package object kafka {
     implicit F: ConcurrentEffect[F],
     context: ContextShift[F]
   ): Resource[F, TransactionalKafkaProducer[F, K, V]] =
-    TransactionalKafkaProducer.resource(settings)
+    new TransactionalKafkaProducer.Builder[F, K, V](settings).withConstantId
```

Comment:
I expect we'll need to rethink the syntax helpers

Reply:
Yes, let's worry about that later on though. :)